src/uct/base/uct_iface.h

```diff
@@ -123,9 +124,13 @@ enum {
                     "UCT_EP_PARAM_FIELD_DEV_ADDR and UCT_EP_PARAM_FIELD_IFACE_ADDR are not defined")
 
 
+#define UCT_EP_PARAM_VALUE(_params, _name, _flag, _default) \
```

Comment:
Are there additional places in UCT code that currently take ep params from field mask, and can be converted to use UCT_EP_PARAM_VALUE?